doc/code/model.rst

```diff
@@ -0,0 +1,3 @@
+.. automodule:: pennylane.model
```

Comment:
Does this now need to be "template"? (Or the other instances below be "model"?)

Reply:
Let me comment in the conversation, I find the name "ansatz" the best choice...

Reply:
Good catch! Fixed.